mmselfsup/models/heads/mae_pretrain_head.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+from mmcv.runner import BaseModule
+
+from ..builder import HEADS
+
+
+@HEADS.register_module()
+class MAEPretrainHead(BaseModule):
+    """Pre-training head for MAE.
+
+    Args:
+        norm_pix_loss (bool): Whether or not normalize target.
+            Defaults to False.
+        patch_size (int): Patch size. Defaults to 16.
+    """
+
+    def __init__(self, norm_pix_loss=False, patch_size=16):
+        super(MAEPretrainHead, self).__init__()
+        self.norm_pix_loss = norm_pix_loss
```

Comment:
norm_pix_loss -> norm_pix

Reply:
Thank you for your valuable comments. `norm_pix` seems more intuitive.